project.clj

```diff
@@ -9,7 +9,7 @@
 
   :dependencies [[org.clojure/clojure "1.9.0"]
                  [org.clojure/core.memoize "0.7.1"]
-                 [com.damballa/abracad "0.4.13"]
+                 [parkside-securities/abracad "0.4.15-SNAPSHOT"]
```

Comment:
Will fix it to 0.4.16. 

I used to be in a culture where all internal libs are all `0.1.0-SNAPSHOT` so that the downstream consumers pick up the latest by them self.

Reply:
@k2n Released a stable version to clojars